WordPress/gutenberg
# global styles slotfill

## What?
Fixes #48068 

## How?
This PR introduces a new SlotFill location in the global styles section.

## Testing Instructions

1. Checkout this branch and build
2. Add the sample code below in your theme or plugin
3. Confirm that slot appears as seen in the screenshot

```
import { registerPlugin } from '@wordpress/plugins';
import { HelpContentSlot } from '@wordpress/edit-site';

// Define a basic component to render the help content inside the slot.
const HelpContent = () => (
	<HelpContentSlot
		renderContent={ () => (
			<p>This is your help content displayed in the slot.</p>
		) }
	/>
);

// Register the plugin.
registerPlugin( 'my-help-content-plugin', {
	render: HelpContent,
} );
```

## Screenshots or screencast 
<img width="284" alt="Global styles" src="https://github.com/user-attachments/assets/c31c5631-c52f-430c-b706-87b4a1bdfdf6">
